web/src/components/changes_review.js

```diff
@@ -15,6 +15,20 @@ import {
 
 import { Line } from 'react-chartjs-2';
 
+function secondsToDhms(seconds) {
```

Comment:
Great, just tried an it work well ! But I think we should use an existing lib for that. Could you try https://momentjs.com/docs/#/durations/ ? There is also an humanize function that seems nice https://momentjs.com/docs/#/durations/humanize/

Reply:
ok I'll try that

Reply:
done